ray_on_golem/provider/node_provider.py

```diff
@@ -311,8 +316,8 @@ def _start_webserver(
     @staticmethod
     def _stop_webserver(ray_on_golem_client: RayOnGolemClient) -> None:
         with cli_logger.group(LOG_GROUP):
-            if not ray_on_golem_client.is_webserver_running():
-                cli_logger.print("Not stopping webserver, as it's already not running")
+            if not ray_on_golem_client.get_webserver_state():
+                cli_logger.print("Not stopping webserver, as it's already not running or shutting down")
```

Comment:
I think we _could_ differentiate the message given the fact that we _know_ whether it's not running at all or just shutting down

```suggestion
            webserver_state = ray_on_golem_client.is_webserver_serviceable()
            if not webserver_state:
                if webserver_state is None:
                    cli_logger.print("Not stopping the webserver, as it's not running.")
                else:
                    cli_logger.print("Not stopping the webserver, as it's already shutting down")
```